python/cudf/cudf/_lib/nvtext/tokenize.pyx

```diff
@@ -15,23 +15,25 @@ from cudf._lib.cpp.nvtext.tokenize cimport (
 )
 from cudf._lib.column cimport Column
 from cudf._lib.scalar cimport DeviceScalar
-
+from cudf import Scalar
```

Comment:
@galipremsagar the issue with this is that it creates a circular import issue. We can't import cuDF globally here. But somehow we need to still do an instance check. I didn't know what to do exactly except import at runtime. Or maybe have a special function somewhere else?

Reply:
Ah, I think then importing where necessary inside a function is fine in this case.

Reply:
After break it seems better to factor this out to cudf.utils.utils. So I opened this comment again.